test/playback.test.js

```diff
@@ -0,0 +1,74 @@
+import QUnit from 'qunit';
+import videojs from 'video.js';
+/* eslint-disable no-unused-vars */
+import { Hls } from '../src/videojs-contrib-hls';
+
+let when = function(element, type, cb, condition) {
+  element.on(type, function func() {
+    if (condition()) {
+      element.off(type, func);
+      cb();
+    }
+  });
+};
+
+let playFor = function(player, time, cb) {
+  let targetTime = player.currentTime() + time;
+
+  when(player, 'timeupdate', cb, () => player.currentTime() >= targetTime);
+};
+
+QUnit.module('Playback', {
+  beforeEach() {
+    QUnit.stop();
+    let video = document.createElement('video');
+
+    video.width = 600;
+    video.height = 300;
+    document.querySelector('#qunit-fixture').appendChild(video);
```

Comment:
Probably not an issue but do we want to remove the element from the DOM between tests?